core/tauri/src/lib.rs

```diff
@@ -14,7 +14,7 @@
 pub use error::Error;
 pub use tauri_api as api;
 pub(crate) use tauri_api::private::async_runtime;
-pub use tauri_macros::*;
+pub use tauri_macros::{command, generate_handler};
```

Comment:
the lib.rs file is pretty big now - but i guess it's ok like this

Reply:
yeah, im hoping in the future to refactor the sealed part somewhere else which would make it look a lot more manageable, but for now this works fine